tests/test_percy_snapshot.py

```diff
@@ -116,44 +94,41 @@ def run_percy_on(self, dir_name):
             if image.endswith('png'):
                 output_name = image.replace('.png', '')
 
-                self.driver.get('http://localhost:8050/{}'.format(image))
+                dash_duo.driver.get('http://localhost:8050/{}'.format(image))
 
-                WebDriverWait(self.driver, 20).until(
-                    EC.presence_of_element_located((By.ID, image))
-                )
+                dash_duo.wait_for_element_by_id(image, 20)
 
                 self.percy_snapshot(
-                    name='{}: {}'.format(dir_name.upper(), output_name)
+                    dash_duo, name='{}: {}'.format(dir_name.upper(), output_name)
                 )
                 time.sleep(2)
 
-    def test_usage(self):
-        # Create and start the app
+    def test_usage(self, dash_duo):
```

Comment:
This whole set of tests would be a great use case for `pytest.mark.parametrize`, eg:
```py
import pytest

@pytest.mark.parametrize('name', ['usage', 'elements', 'layouts', 'style'])
def test_cyps001_snapshots(name, dash_duo):
    app = self.create_app(dir_name=name)
    dash_duo.start_server(app)

    run_percy_on(name, dash_duo)
```